engine-rest/engine-rest/src/main/java/org/operaton/bpm/engine/rest/impl/AuthLogger.java

```diff
@@ -1,9 +1,11 @@
 /*
+ * Copyright 2025 the Operaton contributors.
+ *
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
+ * You may obtain a copy of the License at:
  *
- *     https://www.apache.org/licenses/LICENSE-2.0
+ *     http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
We should use HTTPS Urls. Since I have added it wrong to the Contributing guide and there are a lot of duplications, I have opened https://github.com/operaton/operaton/issues/852 to change that.